src/Storages/MergeTree/MergeTreeDataMergerMutator.cpp

```diff
@@ -79,13 +79,16 @@ void FutureMergedMutatedPart::assign(MergeTreeData::DataPartsVector parts_)
 
     size_t sum_rows = 0;
     size_t sum_bytes_uncompressed = 0;
+    MergeTreeDataPartType future_part_type;
```

Comment:
Maybe better to initialize this value with UNKNOWN?

Reply:
Ok. It's initialized by `UNKNOWN`  by default, but maybe it's a bit confusing.